zkstack_cli/crates/zkstack/src/enable_evm_emulator.rs

```diff
@@ -0,0 +1,56 @@
+use common::{
+    forge::{Forge, ForgeScript, ForgeScriptArgs},
+    spinner::Spinner,
+    wallets::Wallet,
+};
+use config::{forge_interface::script_params::ENABLE_EVM_EMULATOR_PARAMS, EcosystemConfig};
+use ethers::{abi::parse_abi, contract::BaseContract, types::Address};
+use lazy_static::lazy_static;
+use xshell::Shell;
+
+use crate::{
+    messages::MSG_ENABLING_EVM_EMULATOR,
+    utils::forge::{check_the_balance, fill_forge_private_key, WalletOwner},
+};
+
+lazy_static! {
```

Comment:
Nit: `lazy_static` is de-facto obsoleted by `once_cell`. Also, since shell commands are usually executed in isolation, is a static really necessary here? AFAIU, it'll be accessed just once.